src/emcee/ensemble.py

```diff
@@ -164,7 +174,10 @@ def random_state(self):
         so silently.
 
         """
-        return self._random.get_state()
+        try:
```

Comment:
Should this be:
```
if isinstance(self._random, np.random.RandomState):
    return self._random.get_state()
elif isinstance(self._random, Generator):
    return self._random.bit_generator.state
```
Obviously Generator will have to be imported from somewhere, in a similar fashion to what's done in `utils.py`.